keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ComparableLinksTests.kt

```diff
@@ -245,87 +244,88 @@ class ComparableLinksTests : JUnit5Minutests {
           emptyList()
         }
 
-        every { repository.artifactVersions(releaseArtifact) } returns versions.toArtifactVersions(releaseArtifact)
```

Comment:
This data structure is a shortened version of the `ENVIRONMENT_ARTIFACT_VERSIONS` table. As I mentioned above, we pull this once and use this list to avoid querying the db excessively for each version. As a result I think the test is a little easier to read (you can see the "set up" - here version 0 was replaced by version 1, version 1 replaced by version 2, version 2 is current).